plugin/storage/es/spanstore/reader_test.go

```diff
@@ -126,6 +124,7 @@ func withSpanReader(fn func(r *spanReaderTest)) {
 			MaxDocCount:       defaultMaxDocCount,
 		}),
 	}
+	defer closer()
```

Comment:
we talked about where defers should be placed

Reply:
sorry Yuri! I did it inside withSpanReader function call. but How can I defer it? I think I have to modify the closer to use `func()` and `fmt.Errorf`.

Reply:
you create the object in L110, so `defer closer()` should be L111. Always keep defer next to creation.

Reply:
I did that. Just to re-commit.